.eslintrc.js

```diff
@@ -2,7 +2,9 @@ module.exports = {
     parserOptions: {
         requireConfigFile: false
     },
-    'extends': [
-        '@jitsi/eslint-config'
-    ]
+    extends: [ '@jitsi/eslint-config' ],
+    rules: {
+        'no-unused-private-class-members': [ 'error' ],
```

Comment:
We don't use private class members (yet), but this should go on the eslint-config-jitsi repo.